packages/quantic/copy-static-resources.js

```diff
@@ -10,9 +11,21 @@ const main = async () => {
   console.info('Begin copy.');
 
   try {
+    await mkdir(
+      './force-app/main/default/staticresources/coveoheadless/browser/',
+      {recursive: true}
+    );
+    await mkdir(
+      './force-app/main/default/staticresources/coveoheadless/definitions/',
+      {recursive: true}
+    );
+    await copy(
+      './node_modules/@coveo/headless/dist/browser/headless.js',
+      './force-app/main/default/staticresources/coveoheadless/browser/headless.js'
+    );
     await copy(
-      './node_modules/@coveo/headless/dist',
-      './force-app/main/default/staticresources/coveoheadless'
+      './node_modules/@coveo/headless/dist/definitions',
```

Comment:
Are the definitions needed?

Reply:
Yeah, they are used throughout Quantic project